boards/FSM/Test/Src/Test_StateMachine.cpp

```diff
@@ -103,13 +123,56 @@ class FsmStateMachineTest : public testing::Test
                                                   App_GetAirClosedState() };
     }
 
+    void CheckInRangeCanSignalsInAllStates(
+        float  min_in_range,
+        float  max_in_range,
+        float &fake_in_range,
+        float (*in_range_can_signal_getter)(const struct FsmCanTxInterface *),
+        uint8_t (*out_of_range_can_signal_getter)(
+            const struct FsmCanTxInterface *),
```

Comment:
The variable names didn't make sense to me. We're describing the values not the in-range checks.
```suggestion
        float  min_value,
        float  max_value,
        float &fake_value,
        float (*value_can_signal_getter)(const struct FsmCanTxInterface *),
        uint8_t (*out_of_range_can_signal_getter)(
            const struct FsmCanTxInterface *),
```